src/Microsoft.Health.Dicom.Api/Controllers/ExtendedQueryTagController.cs

```diff
@@ -43,20 +46,21 @@ public ExtendedQueryTagController(IMediator mediator, ILogger<ExtendedQueryTagCo
             _featureEnabled = featureConfiguration.Value.EnableExtendedQueryTags;
         }
 
-        [BodyModelStateValidator]
-        [ProducesResponseType(typeof(JsonResult), (int)HttpStatusCode.Accepted)]
         [HttpPost]
+        [BodyModelStateValidator]
+        [ProducesResponseType(typeof(AddExtendedQueryTagResponse), (int)HttpStatusCode.Accepted)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         [Route(KnownRoutes.ExtendedQueryTagRoute)]
         [AuditEventType(AuditEventSubType.AddExtendedQueryTag)]
-        public async Task<IActionResult> PostAsync([FromBody] IEnumerable<AddExtendedQueryTagEntry> extendedQueryTags)
+        public async Task<IActionResult> PostAsync([Required][MinLength(1)][FromBody] IReadOnlyCollection<AddExtendedQueryTagEntry> extendedQueryTags)
```

Comment:
I'm not sure what error we would have here if length is 0, but we do give proper error message here: https://github.com/microsoft/dicom-server/blob/b7f8f0ec2cb2969920f2adbc1f2ae69ca616e432/src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagEntryValidator.cs#L55

Reply:
+1, lets make sure proper status code is set